### PR TITLE
whitelisting

### PR DESCRIPTION
**Pull request by Piyushhbhutoria**

### CODE REVIEW

File: VWOSdk/Core/VWOClient.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@
 #pragma warning restore 1587
 
 using System.Collections.Generic;
+using System;
```
Comment (sahilbathla) [inline VWOSdk/Core/VWOClient.cs]: do we really need System , can we not import specific namespace?

File: VWOSdk/Core/VWOClient.cs [hunk 2]
```diff
@@ -384,11 +422,49 @@ private UserAllocationInfo AllocateVariation(string campaignKey, string userId,
...
+                    variationTargettingVariable = new Dictionary<string, dynamic>();
+                    variationTargettingVariable.Add("_vwo_user_id", userId);
+                }
+                List<Variation> whiteListedVariations = this.GetWhiteListedVariationsList(userId, campaign, campaignKey, customVariables, variationTargettingVariable);
+
+                return this._variationAllocator.TargettedVariation(userId, whiteListedVariations);
+            }
+            return null;
+        }
+        private List<Variation> GetWhiteListedVariationsList(string userId, BucketedCampaign campaign, string campaignKey, Dictionary<string, dynamic> customVariables, Dictionary<string, dynamic> variationTargettingVariable)
```
Comment (sahilbathla) [inline VWOSdk/Core/VWOClient.cs:440]: new line before new function

**Review by sahilbathla — CHANGES_REQUESTED** [review]
Update Readme 
Update Example app

File: VWOSdk/Core/VWOClient.cs (hunk 1 above)
Comment (shubhamgupta14cse) [reply VWOSdk/Core/VWOClient.cs]: Not anymore, need it for console. shall remove now.

File: VWOSdk/Core/VWOClient.cs (hunk 2 above)
Comment (shubhamgupta14cse) [reply VWOSdk/Core/VWOClient.cs:440]: Done.

**Review by sahilbathla — APPROVED** [review]

**Pull request merged**